### DISCUSSION TITLE
Chain sync slow after crash

### DISCUSSION THREAD

**Discussion by ambergroup-io** [filecoin-project/lotus#5652] [Build and Sync] [answered]
**Describe the bug**
The node is started on December 20, 2020. It exited abnormally on February 13, 2021.
I restarted it on the 15th. It syncs very slowly. An average of 30-40 seconds per block.
I found that it keeps reading a lot of data from disk. About 70M/s.

I remove ".lotus/datastore/chain", then use snapshots, about 9 seconds per block. The desk io is about 500K/s.

**Version (run `lotus version`):**
$ lotus version
Daemon:  1.4.0+git.bb5a92e2f+api1.0.0
Local: lotus version 1.4.0+git.bb5a92e2f

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
what kind of hardware is the node running on?

**Comment by ambergroup-io** [comment]
> what kind of hardware is the node running on?

Amazon Linux 

$ lscpu
Architecture:        x86_64
CPU op-mode(s):      32-bit, 64-bit
Byte Order:          Little Endian
CPU(s):              4

$ free -m -h
              total        used        free      shared  buff/cache   available
Mem:            15G        6.4G        157M        146M        8.8G        9.1G

$ df -h
Filesystem      Size  Used Avail Use% Mounted on
devtmpfs        7.7G     0  7.7G   0% /dev
tmpfs           7.7G  147M  7.6G   2% /run
tmpfs           7.7G     0  7.7G   0% /sys/fs/cgroup
/dev/nvme0n1p1  200G   35G  166G  18% /
/dev/nvme1n1    2.0T  673G  1.3T  36% /data   --------------file node(No other programs use it)
tmpfs           1.6G     0  1.6G   0% /run/user/1001

When the node synchronization is slow, the disk is used 1.3T.

Now, the disk read io is 10M/s. (The node runs for 5 days.)

**Reply by Sakuya-CN** [reply]
How long does it take you to synchronize

**Reply by hcm609** [reply]
Did you manage to solve this somehow?

**Comment by ambergroup-io** [comment]
We are just as observers, we are just verifying the most recent
transactions. We have updated the version. We delete the data regularly to
solve this problem.
When is there a light node version.

hcm609 ***@***.***> 于2021年11月15日周一 上午3:05写道：

**Comment by LexLuthr** [comment, marked as answer]
@ambergroup-io The lotus lite node is out now. Do you still need help with this?
https://lotu.sh/docs/set-up/lotus-lite/
